platform/darwin/test/MGLStyleTests.mm

```diff
@@ -100,15 +100,6 @@ - (void)testVersionedStyleURLs {
     XCTAssertEqualObjects([MGLStyle satelliteStreetsStyleURLWithVersion:99].absoluteString,
                           @"mapbox://styles/mapbox/satellite-streets-v99");
 #pragma clang diagnostic push
```

Comment:
The `push`es and `pop`es can be removed, too — they served to limit the scope of the ignored warnings.